lib/utils.js

```diff
@@ -57,6 +57,16 @@ utils.sanitize = function sanitize(str) {
 };
 
 /**
+ * Sanitize a key by removing spaces or reserved charaters
+ *
+ * @param {String} str
+ */
+
+utils.removeSpaces = function removeSpaces(str) {
```

Comment:
Only comment is that the function name is not strictly correct. Perhaps a better name is `replaceSpaces`, and then allowing an optional argument for the replacement. For example:

```js
/**
 * Replaces whitespace with a character or string.
 *
 * @param {string} str - source string
 * @param {string} [replacement] - character or string to replace spaces with. Default: "_"
 * @return {string}
 */
utils.removeSpaces = function removeSpaces(str, replacement) {
  return typeof str === 'string' ? str.replace(/\s/g, replacement || '_') : str;
};
```